tools/nl/embeddings/README.md

```diff
@@ -4,36 +4,46 @@ This directory contains the data sheets (containing StatVar DCID and
 descriptions) and script used to construct the Stat Var Embeddings Index that
 is loaded into the NL Server in Website.
 
-## Latest Embeddings Google Sheet
+There are multiple sizes of indexes: small, medium.
+
+As of May 2023, the small index has ~1.3K variables and medium index has 5.3K
+variables.
+
+## Curated Input Google Sheet
 
 Latest sheet as of May 2023 is
 [`DC_NL_SVs_Curated`](https://docs.google.com/spreadsheets/d/1-QPDWqD131LcDTZ4y_nnqllh66W010HDdows1phyneU).
 
+This is a common sheet across the different index sizes.
+
 ## Making a change to the embeddings.
 
 1. Make edits to the [latest sheet](https://docs.google.com/spreadsheets/d/1-QPDWqD131LcDTZ4y_nnqllh66W010HDdows1phyneU). The columns in this sheet are:
 
-* `dcid`: the StatVar DCID.
-* `Name`: the name of the StatVar
-* `Description`: the description(s) of the StatVar. Multiple descriptions are acceptable. If multiple description strings are provided, they must be semi-colon delimited. This column can be expected to be used as part of any automated alternative generation processed, e.g. using an LLM.
-* `Override_Alternatives`: if this column has any value in a row, all other columns are ignored. Multiple strings are acceptable for this column. If multiple strings are provided, they must be semi-colon delimited.
-* `Curated_Alternatives`: a semi-colon delimited list of strings which can serve as alternative ways of referring to the StatVar.
+   * `dcid`: the StatVar DCID.
+   * `Name`: the name of the StatVar
+   * `Description`: the description(s) of the StatVar. Multiple descriptions are acceptable. If multiple description strings are provided, they must be semi-colon delimited. This column can be expected to be used as part of any automated alternative generation processed, e.g. using an LLM.
+   * `Override_Alternatives`: if this column has any value in a row, all other columns are ignored. Multiple strings are acceptable for this column. If multiple strings are provided, they must be semi-colon delimited.
+   * `Curated_Alternatives`: a semi-colon delimited list of strings which can serve as alternative ways of referring to the StatVar.
+
+1. Add any auto-generated input CSVs to `data/autogen_input/<index-size>/`.
```

Comment:
It took me a while to figure out that this is the file (or process) through which the small vs medium are distinguished. If, theoretically, the same file was provided under `data/autogen_input/small/` and `data/autogen_input/medium/` then the embeddings would be the same size, right? So we should make it clear in this README.md that this is the file/process through which the number/size of the SVs/Embeddings is primarily impacted. The rest of the inputs are all common across the embeddings sizes.

Reply:
thanks, clarified.